autoload/ctrlp.vim

```diff
@@ -953,7 +953,7 @@ fu! s:MapSpecs()
 	if !( exists('s:smapped') && s:smapped == s:bufnr )
 		" Correct arrow keys in terminal
 		if ( has('termresponse') && v:termresponse =~ "\<ESC>" )
-			\ || &term =~? '\vxterm|<k?vt|gnome|screen|linux|ansi|tmux'
+			\ || &term =~? '\vxterm|<k?vt|gnome|screen|linux|ansi|tmux|st(-\w+)?'
```

Comment:
Seems good but too generic word. For example, if starbucks :coffee:  create new terminal named `starbucks`, it will be matched. :(
please add `$` after `?`.